parcel.go

```diff
@@ -14,47 +15,88 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) "+
+		"VALUES (:client, :status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.CreatedAt))
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Лучше в модулях не делать вывод, а использовать оборачивание ошибки, чтобы вызывающая сторона сама решила, что с ней делать

```suggestion
		return fmt.Errorf("Error in INSERT: : %w", err)
```